content/basics/basics.mdx

```diff
@@ -0,0 +1,327 @@
+---
+name: Basics
+route: /basics
+---
+
+import { Playground } from 'docz'
+
+# Basic Introduction
+
+[MobX](https://mobx.js.org/README.html) lets you create observable state. This
+library, `mobx-react` integrates it with React. It provides a way for you to
+create React components that respond to MobX state changes.
+
+> Frameworks such as [mobx-state-tree](https://mobx-state-tree.js.org) and
+> [mobx-keystone](https://mobx-keystone.js.org/) build on
+> top of MobX to offer a higher-level way to create observable models. These models
+> give you additional properties, such as automatic serialization to JSON. This
+> introduction uses plain MobX models. The principles of React
+> integration are the same for these advanced libraries are the same
+> as they are also built with MobX.
+
+## MobX models
+
+Let's consider MobX models for animals in a zoo, where each animal
+has a certain amount of energy. If the energy is below 50, the animal
+is hungry. Animals can be fed to increase their energy levels:
+
+> These code examples use TypeScript. If you use plain JavaScript you
+> can skip the type declarations.
+
+```tsx
+import { observable, action, computed } from 'mobx'
+
+class Animal {
+  name: string
+  species: string
+  @observable energy: number
```

Comment:
Given that upcoming MobX 6 is dropping support for decorators, let's not spread it here. Check https://github.com/mobxjs/mobx/issues/2325 for API examples that are most likely set in the stone by now.

Reply:
Okay, I didn't know that and I need to look into that, thanks!

Reply:
Continuing here with [your concerns about decorators](https://github.com/mobxjs/mobx-react-docz/pull/102#issuecomment-646591780). I am not sure what is your confusion. The first comment shows "API changes" section on how it will look like with MobX 6. I know it's not technically available yet, but there is plenty of examples with decorators in other docs. Here I would like to target it more toward the future. We can even include some warning this API is not available yet. We can have it here like "promotion" on what's will become "next standard"

Reply:
I am not interested in documenting an API that isn't available yet.